src/signer.ts

```diff
@@ -288,6 +315,22 @@ export class Signer {
     }
   }
 
+  private handleOwnerInitialized({data: msgData, origin}: SignerMessageEvent): {
+    initialized: boolean;
+  } {
+    try {
+      // Use Zod to assert that the owner is not nullish and not anonymous.
```

Comment:
- I think we can space this comment.
- No strong opinion but for code consistency, should we use `safeParse` here?